G2P.pm

```diff
@@ -64,13 +64,14 @@ limitations under the License.
                          allele frequencies. Allele frequencies are retrieved from VCF files. If
                          af_from_vcf is set to 1 but no VCF collections are specified with --af_from_vcf_keys
                          all available VCF collections are included. 
-                         Available VCF collections: topmed, uk10k, gnomADe, gnomADg, gnomADg_r3.0
-                         Separate multiple values with '&'
+                         Available VCF collections: topmed, uk10k, gnomADe, gnomADe_r2.1.1, gnomADg, gnomADg_v3.1.2.
+                         Separate multiple values with '&'.
                          VCF collections contain the following populations: 
-                         topmed: TOPMed
-                         uk10k: ALSPAC, TWINSUK
-                         gnomADe: gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS
-                         gnomADg: gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH
+                         topmed_GRCh37 & topmed_GRCh38 : TOPMed
+                         uk10k_GRCh37 & uk10k_GRCh38 : ALSPAC, TWINSUK
+                         gnomADe_GRCh37 & gnomADe_r2.1.1_GRCh38: gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS
+                         gnomADg_GRCh37 & gnomADg_v3.1.2_GRCh38: gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH
+                         Need to use af_from_vcf paramter to use this option. 
```

Comment:
Including the assembly in the key is confusing.
Do you think something like this would be clear?

```
topmed: TOPMed (available for GRCh37 and GRCh38)
uk10k: ALSPAC, TWINSUK  (available for GRCh37 and GRCh38)
gnomADe: gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS
  use 'gnomADe' for assembly GRCh37 and 'gnomADe_r2.1.1' for assembly GRCh38
gnomADg: gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH
  use 'gnomADg' for assembly GRCh37 and 'gnomADg_v3.1.2' for assembly GRCh38
```